docs/docs/04-features.md

```diff
@@ -125,56 +123,36 @@ Note that this only works for directories that have been mounted via a `mount:*`
 
 ### JSX
 
-#### Compile to JavaScript
-
+Snowpack has built-in support to handle `.jsx` & `.tsx` source files in your application. 
 
-Snowpack automatically builds all `.jsx` & `.tsx` files to JavaScript during development and production builds. 
+**Note: Snowpack's default build does not support JSX in  `.js`/`.ts` files.** If you can't use the `.jsx`/`.tsx` file extension, you can create a Babel config file and use the official Babel plugin to build your application:
 
-**Note: Snowpack's default build supports JSX with both React & Preact as long as a React/Preact import exists somewhere in the file.** To set a custom JSX pragma, you can configure our default esbuild yourself:
 
 ```js
 // snowpack.config.json
-// Optional: Define your own JSX factory/fragment
+// [npm install --save-dev @snowpack/plugin-babel]
 {
-  "scripts": {
-    "build:tsx": "esbuild --jsx-factory=h --jsx-fragment=Fragment --loader=tsx"
-  }
-}
-```
-
-**Note: Snowpack's default build does not support JSX in  `.js`/`.ts` files.** You'll need to define your own build script to support this. You can define your own JSX->JavaScript build step via a [Build Script integration](#build-scripts).
-
-```js
-// snowpack.config.json
-// Optional: You can define your own JSX build step if you'd like.
-{
-  "scripts": {
-    "build:jsx": "babel --filename $FILE",
-  }
+  "plugins": ["@snowpack/plugin-babel"]
 }
 ```
 
 ### TypeScript
 
 #### Compile to JavaScript
 
-Snowpack automatically builds all `.ts` & `.tsx` files to JavaScript. Snowpack will not perform any type checking by default (see below), only building from TS->JS.
-
-You could also choose to define your own TSX->JavaScript build step via a [Build Script integration](#build-scripts).
+Snowpack has built-in support to handle `.ts` & `.tsx` source files in your application. If you prefer to use Babel, you can instead create a Babel config file and use the official Babel plugin to build your application:
```

Comment:
What are your thoughts for bubbling up this file list to the top of the **Features** section? Something short and sweet like:

```
## Features

### Filetype Support

Snowpack can support the following filetypes with no configuration:

- JavaScript (`.js`, `.jsx`)
- TypeScript (`.ts`, `.tsx`)
- CSS (`.css`) / CSS Modules (`.module.css`)
- Images (`.svg`, `.jpg`, `.png`, etc.)

To customize behavior, or to import other files (`.svelte`, `.vue`), keep reading.
```

Not too fixed on the format, but I think an “early return” at the top of this section may be helpful for users to see if they need to configure Snowpack or add plugins or not.

Reply:
Unrelated: given this copy change, what are your thoughts on renaming this section to “Using Babel”?

Reply:
Took a pass at this, lmk what you think. It definitely cleans up a fair bit of duplication between this section and "Recipes" below.